### ISSUE TITLE
조도복원(딥러닝) 개선

### ISSUE THREAD

**Issue by sick2024471** [meo-yeong/derain_dedrop_dehaze#2]
adjustment, train구조를 개선시켜 조도복원 모델 개선시도
adjustment.py같은 경우는 Skip Connection구조, encoder, decoder 구조를 추가 하여 저주파, 고주파를 함께 보존

train.py같은경우
기존에는 단순 L1 LOSS였으나  
L_decom(Decomposition Loss), L_enhance(Enhancement Loss), L_ir(illumination Smoothness Loss) 추가하여
Loss = L_decom + L_enhance + Lir로 학습 강화 

<img width="1001" height="466" alt="Image" src="https://github.com/user-attachments/assets/65697608-8fbf-4331-8c2d-779fb7d16d75" />
위 사진은 결과 예시이며 밝기는 확실히 개선됬으나 화질이 안좋아지고, 색왜곡이 조금 있음(이것도 다시 한번 알아봐야됨)

+α으로 한번 만들어 본것으로 화질. 색왜곡문제 때문에 아마 이것까지 추가하기엔 많이 늦었을것으로 예상...

참고 코드
https://github.com/weichen582/RetinexNet 
이게 논문에서 바로 이어진 코드, TensorFlow로 구현됨(공식)

https://github.com/aasharma90/RetinexNet_PyTorch
이건 RetinexNet을 Pytorch로 구현한 버전(비공식)
